Azure-Samples/dotnetcore-sqldb-ghactions
# added project and workflows

## Purpose
Added the full code sample and workflows needed to deploy .netcore + database. 

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ x] Other... Please describe: full code sample
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
make a code change
git push [repo-name]
this triggers the workflows: check the workflows for needed GitHub secrets
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* A web app and database should get deployed in your Azure environment

## Other Information
